lib/client.js

```diff
@@ -142,17 +142,17 @@ Client.prototype.eachRow = function(query, params, options, rowCallback, endCall
     this._execute(args.query, args.params, args.options, pageCallback, endCallback);
 };
 
-Client.prototype.new_query = function(query, callback) {
-    return this.client.new_query();
+Client.prototype.query = function(query, callback) {
```

Comment:
It's a little confusing to use `query` as a noun here, don't you think? I would think of `client.query` as telling the client to issue a query, not give me some query object. Also what are the arguments to this function for?